lib/memset.cpp

```diff
@@ -0,0 +1,8 @@
+#include <fried/string.h>
+
+void __attribute__((weak)) *memset(void *bufptr, unsigned char value, unsigned long long size) {
+	unsigned char* buf = (unsigned char*) bufptr;
+	for (size_t i = 0; i < size; i++)
+		buf[i] = (unsigned char) value;
```

Comment:
```suggestion
		buf[i] = value;
```
이 부분은 캐스팅이 필요 없어 보입니다.

Reply:
https://github.com/Hhro/lets_make_kernel/commit/c7558fa7a3c04962ceac58c3666576bacb163a42#diff-f389395629f969f547ab53247a9c7889R6